### DISCUSSION TITLE
Connect with Azure AD SSO returns status code: 500 "oops, something went wrong"

### DISCUSSION THREAD

**Discussion by loic667** [requarks/wiki#4798] [Error / Bug Report]
Hello,

We had our Azure SSO login working fine for several months but since the last update to version 2.5.255 we can't login or logout through the Azure SSO. We didn't touch any configuration on the Azure SSO side.
This is what we get. "oops, something went wrong; invalid authentication provider". Status code: 500

We double checked configuration on Azure AD and on Wikijs, and all looks fine. 

Any idea on what we could do?
Thanks! 

<img width="425" alt="image" src="https://user-images.githubusercontent.com/62846494/146570702-1dec4898-c8d9-4c18-98f3-ea4488f0ef25.png">

**Comment by deeepakkumarthakur16** [comment]
@NGPixel I am also facing the same issue. Can you please let us know if there is any fix for this available?

**Comment by aletsch** [comment]
This is a late response but maybe this will help someone. I had the same problem, in the browser i had a 500 response and on the server side in the docker wiki log i had a:
```
2024-12-16T09:31:45.343Z [MASTER] error: Authentication Strategy Azure Active Directory (XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXX): [ FAILED ]
2024-12-16T09:31:45.343Z [MASTER] error: Invalid value for redirectUrl.The URL must be valid and be https://
```

On the Configuration of the Wiki i had set the url as "http://wiki.myurl.com" which resulted in a displayed callback url of `http` instead of `https`. But MS Entra only allows `https` callback URLs so i set `https` there, which was true when seen from the outside/MS Entra (behind reverse proxy with ssl). But this error may have come by the side from the Entra Implementation on the side of the wiki and not of Entra itself i suppose.

When i changed that to https it worked like it should. It is a simpel but hard to find issue.
